api/app/Models/User.php

```diff
@@ -1,4 +1,4 @@
-<?php
```

Comment:
Accidental change?

Reply:
I just ran pint and put my confidence in it... guess my confidence was misplaced :shrug: